nifi-nar-bundles/nifi-standard-bundle/nifi-standard-processors/src/test/java/org/apache/nifi/processors/standard/TestListenUDP.java

```diff
@@ -173,7 +170,7 @@ public void testRunWhenNoEventsAvailable() {
 
     @Test
     public void testWithSendingHostAndPortSameAsSender() throws IOException, InterruptedException {
-        final Integer sendingPort = NetworkUtils.getAvailableUdpPort();
+        final int sendingPort = 27911;
```

Comment:
Will this hard-coded port result in intermittent failures? Can it be set to `0` for constructing the `DatagramSocket`?

Reply:
Good catch, I meant to come back to that one, as I wasn't 100% sure what was happening there. Will address, though.